### PR TITLE
Add CRTFilter

### PR DESCRIPTION
**Pull request by finscn**
the effect is : 
image

Online example: http://fatidol.com/pixi-filters/

### CODE REVIEW

File: bundle/README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Filters include:
 * **ConvolutionFilter** _@pixi/filter-convolution_
 * **CrossHatchFilter** _@pixi/filter-cross-hatch_
 * **DotFilter** _@pixi/filter-dot_
+* **CRTFilter** _@pixi/filter-crt_
```
Comment (bigtimebuddy) [inline bundle/README.md]: Could you alphabetize?

File: filters/crt/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2013-2017 Mathew Groves, Chad Engler
```
Comment (bigtimebuddy) [inline filters/crt/LICENSE]: Copyright update?

File: filters/crt/src/CRTFilter.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+ * @param {number} [options.vignettingAlpha=1.0] - Amount of opacity of vignette
+ * @param {number} [options.vignettingBlur=0.3] - Blur intensity of the vignette
+ * @param {number} [time=0] - TODO
+ */
+export default class CRTFilter extends PIXI.Filter {
+    constructor(options) {
+        super(vertex, fragment);
+
+        Object.assign(this, {
+            curvature: 1.0,
```
Comment (bigtimebuddy) [inline filters/crt/src/CRTFilter.js]: Each property needs documentation.

File: filters/crt/src/crt.frag [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        float outter = SQRT_2 - vignetting * SQRT_2;
+        float darker = clamp((outter - length(dir) * SQRT_2) / ( 0.00001 + vignettingBlur * SQRT_2), 0.0, 1.0);
+        rgb *= darker + (1.0 - darker) * (1.0 - vignettingAlpha);
+    }
+
+    gl_FragColor.rgb = rgb;
+
+    // float m = max(0.0, 1.0 - 2. * max(abs(uv.x), abs(uv.y)) / _c);
+    // float alpha = min(m * 200., 1.);
+    // gl_FragColor = vec4(rgb / gl_FragColor.a * alpha, alpha);
```
Comment (bigtimebuddy) [inline filters/crt/src/crt.frag]: Could you remove commented out lines?

**Comment by bigtimebuddy** [comment]
My only API criticism would be `vignetting` might be better as the noun `vignette`. Similarly, `vignettingBlur` and `vignettingAlpha` should be `vignetteBlur` and `vignetteAlpha`.

**Comment by finscn** [comment]
@bigtimebuddy  `vignetting` also could be a `noun`.
And it 's different from `vignette`

**Comment by bigtimebuddy** [comment]
Okay. I've used those terms interchangeably, but I'm fine leaving it as it. Thanks.

**Pull request merged**